mongocrypt/build.gradle.kts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2019-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import de.undercouch.gradle.tasks.download.Download
+import java.io.ByteArrayOutputStream
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+    }
+    dependencies {
+        "classpath"(group = "net.java.dev.jna", name = "jna", version = "5.11.0")
+    }
+}
+
+plugins {
+    // Needed to download libmongocrypt from s3.
+    id("de.undercouch.download") version "5.6.0"
+}
+
+group = "org.mongodb"
+base.archivesBaseName = "mongodb-crypt"
+description = "MongoDB client-side crypto support"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    api(project(path = ":bson", configuration = "default"))
```

Comment:
The latest version of BSON is now used, as opposed to the version specified in the libmongocrypt repository. For reference, see the previous dependency listed here: [libmongocrypt build.gradle.kts](https://github.com/mongodb/libmongocrypt/blob/f9adce53f6e676cb226b33b411501fe0b6faeca0/bindings/java/mongocrypt/build.gradle.kts#L61).